packages/appium/lib/appium.js

```diff
@@ -108,10 +108,13 @@ class AppiumDriver extends DriverCore {
 
     // allow this to happen in the background, so no `await`
     // catch this to avoid an unhandled rejection
```

Comment:
`make sure we catch any possible errors to avoid unhandled rejections`

Reply:
not sure what the issue is?

Reply:
we are not catching "this" anymore. now we are using a catch inside the IIFE async function. i was giving a suggestion for rewording the comment to apply more accurately.

Reply:
nvm, updated the comment